Makefile

```diff
@@ -4,29 +4,31 @@ GOBIN = go
 # i.e. ignore local $GOPATH/src installed sources
 GOPATH_TMP = $(CURDIR)/.GOPATH
 GO = /usr/bin/env GOPATH=$(GOPATH_TMP) $(GOBIN)
-GO_FLAGS =
 GOFMT = gofmt
 VERSION = dev-$(shell date +%FT%T%z)
 
-OS = linux
-ARCH = amd64
+GO_OS = GOOS="${GOOS}"
```

Comment:
This is weird. Why not just:
```make
export GOOS
export GOARCH
```

(ditto `CC`, `CXX`, etc below)

Reply:
Yep I'll remove it. Actually that was used for debugging purpose